engine/client/cl_main.c

```diff
@@ -2473,7 +2473,7 @@ qboolean CL_PrecacheResources( void )
 			{
 				if( FBitSet( pRes->ucFlags, RES_WASMISSING ))
 				{
-					Con_Printf( S_ERROR "%s%s couldn't load\n", DEFAULT_SOUNDPATH, pRes->szFileName );
```

Comment:
Regular user does not know what is precache, it need only know that it failed to load